### DISCUSSION TITLE
Table style header

### DISCUSSION THREAD

**Discussion by MINT-KISS** [Live-Charts/LiveCharts2#1737] [Q&A] [answered]
I am using livecharts2 in wpf. I made a gantt chart using 2 stackedrowseries (one with transparent rows to indent the edge, the other with the values themselves), and now I want to make a header for it like this
![Pic](https://github.com/user-attachments/assets/8a13d8b7-44dd-44c0-9117-55cffb75a97f)

The problem is that I need different length blocks for months and years, so I can't just use ColumnSeries.

I tried to do it with VisualElement, but I ran into the fact that there is no rectangle with center aligned text inside. LabelVisual doesn't have a stroke, so it's unclear where one month ends and another begins.

I need the header to support zooming and scrolling. To keep it visible I use a separate YAxis for it.

**Comment by MINT-KISS** [comment]
I have studied the source code of GeometryVisual class and noticed that it sets vertical and horizontal alignment for text Align.Start.
@beto-rodriguez can you please add possibility to change these parameters?

**Reply by beto-rodriguez** [reply, marked as answer]
Yes, that (and much more) is included in #1705, that PR is huge improvement in the library to create custom drawn shapes in the canvas/chart.

**Reply by MINT-KISS** [reply]
Thanks! I got it
